content/en/blog/_posts/2021-07-09-How-to-choose-a-SIG-as-a-non-code-Kubernetes-contributor.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: blog
+title: "How to choose a SIG as a non-code Kubernetes contributor"
+linkTitle: "How to choose a SIG as a non-code Kubernetes contributor"
+Author: Chris Short
+date: 2021-07-09
```

Comment:
We'll want to update the date, maybe monday of next week? 7/19?
It should also be reflected in the file name 👍

Reply:
thanks, will update that.